multipass/Linux.md

```diff
@@ -251,25 +251,25 @@ Memory usage:   259.7MiB out of 3.8GiB
 Mounts:         --
 ```
 
-Let's take the IP address starting with "10" and paste it into our browser, then add a colon and the portainer default port, 9000, like this: 10.115.5.235:9000. This will take us to the Portainer login page, where we can set a username and password.
+Copy the IP address starting with "10" and paste it into your browser, then add a colon and the portainer default port, 9000. It should look like this: 10.115.5.235:9000. This will take you to the Portainer login page where you can set a username and password.
 
 ![|720x543](https://assets.ubuntu.com/v1/75a164a1-mp-linux-5.png) 
 
-From there, let's select a local docker environment.
+From there, select a local docker environment.
 
 ![|720x601](https://assets.ubuntu.com/v1/ee3ff308-mp-linux-6.png) 
 
-From there, we can click into the newly created "local" docker endpoint, navigate to the app templates page, and select NGINX
+Inside the newly selected local docker environment, navigate to the table of content, click on "app templates", and select NGINX.
```

Comment:
Thanks for the nitpicks. Can't use the ">" method because of the variety of actions needed in the steps (the style guide seems to specifically warn against this)